asserts/system_user.go

```diff
@@ -309,5 +343,13 @@ func systemUserFormatAnalyze(headers map[string]interface{}, body []byte) (forma
 		formatnum = 1
 	}
 
+	presence, err := checkOptionalString(headers, "user-presence")
+	if err != nil {
+		return 0, err
+	}
+	if presence != "" {
+		formatnum = 2
```

Comment:
maxSupportedFormat setting up in asserts.go and corresponding tests need also to change

Reply:
I updated this, the interesting thing is the tests are set not to fail if the version is above what they expect which is interesting.